client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -3,23 +3,87 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import map from 'lodash';
+import Button from 'components/button';
+import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
 
 class JetpackOnboardingSummaryStep extends React.PureComponent {
+	renderCompleted = () => {
+		const { translate } = this.props;
+		const stepsCompleted = [
+			translate( 'Site title & description' ),
+			translate( 'Type of Site' ),
+			translate( 'Type of Homepage' ),
+			translate( 'Contact Us Form' ),
+			translate( 'Jetpack Connection' ),
+		];
+
+		return Object.keys( stepsCompleted ).map( stepCompleted => (
+			<div className="steps__summary-column entry completed">
+				<Gridicon icon="chevron-down" size={ 12 } />
+				{ stepsCompleted[ stepCompleted ] }
+			</div>
+		) );
+	};
+
+	renderTodo = () => {
+		const { translate } = this.props;
+		const stepsTodo = [
+			translate( 'Choose a Theme' ),
+			translate( 'Add a Site Address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Site Address', 'NULL', { context: 'NULL'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-onboarding-summary-include-content)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).